includes/Modules/Analytics.php

```diff
@@ -134,6 +134,16 @@ function() {
 				$this->handle_provisioning_callback();
 			}
 		);
+
+		// Add the timezone data for users provisioning new accounts.
+		add_filter(
+			'googlesitekit_admin_data',
+			function( $admin_data ) {
+				$selected_zone          = get_option( 'timezone_string' );
+				$admin_data['timezone'] = $selected_zone;
+				return $admin_data;
+			}
+		);
```

Comment:
Let's remove this and instead expose it on `_googlesitekitBaseData`, so that the `core/site` store can then expose it via a `getTimezone()` selector. Also, this should probably happen right in the main function in the `Assets` class, this value is nothing specific to Analytics.

Reply:
👍 I'll try changing this.

Reply:
Moved data to datastore in https://github.com/google/site-kit-wp/pull/1389/commits/4de6bc6e7506c466f74f3dae27076daace5db6a1 and https://github.com/google/site-kit-wp/pull/1389/commits/63e53c28b307b1ead697a8c36b575bf50b622476